scene/gui/video_stream_player.cpp

```diff
@@ -237,6 +237,12 @@ bool VideoStreamPlayer::has_loop() const {
 void VideoStreamPlayer::set_stream(const Ref<VideoStream> &p_stream) {
 	stop();
 
+	// Make sure to handle stream changes seamlessly, e.g. when done via
+	// translation remapping.
```

Comment:
Shouldn't this comment be below?

Reply:
It was meant to cover both at once, but since they're split it's not very clear yeah.